inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -269,3 +270,38 @@ def insert_wf_record_source(json, record_uuid, source):
     else:
         record_source.json = json
     db.session.commit()
+
+
+def resolve_validation_callback_url():
+    """Resolve validation callback.
+
+    Returns the callback url for resolving the validation errors.
+
+    Note:
+        It's using ``inspire_workflows.callback_resolve_validation``
+        route.
+    """
+    return url_for(
+        'inspire_workflows.callback_resolve_validation',
+        _external=True
+    )
+
+
+def validation_errors(data, schema):
```

Comment:
you are missing a verb here too xd

Reply:
Hmm I didn't want to have duplicate the names (``get_validation_errors``), updated!

Reply:
you can use `make_validation_errors` or `create_validation_errors` or similar too, get is ok for me though (as we discussed, the verb is not that much of a deal, there's lots of synonyms).

Reply:
with ‘’as _get_validation_errors’’, IMO makes it more consistent. WDYT?